javascript/javascript1/week3/javascript.js

```diff
@@ -0,0 +1,130 @@
+// Remove name from array list
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+];
+const nameToRemove = "Ahmad";
+for (var i = names.length - 1; i >= 0; i--) {
```

Comment:
Nice work , and it is just my suggestion ,you can also write for loop in easy way like for (var i = 0; i <names.length; i++)

Reply:
huhmm... thats right, I made little complicated. I just updated.